dev-env

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+set -eu
+
+declare -a POSSIBLE_DOCKER_SOCK_PATHS=(
```

Comment:
I used array because it was easier and cleaner than two constants.

Reply:
```suggestion
# This is an array
declare -a POSSIBLE_DOCKER_SOCK_PATHS=(
```

(this is some advanced bash here ^^)